addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -536,6 +552,26 @@ export class Rtc extends Record {
         this.setTalking(true);
     }
 
+    async openPip() {
+        if (this.isHost) {
+            await this.pipService.openPip();
+            return;
+        }
+        this.notification.add(_t(UNAVAILABLE_AS_REMOTE), {
+            type: "warning",
+        });
+        // Cannot reliably activate pip mode from a remote tab as it requires transient activation.
+        // this._remoteAction({ pip: true });
+    }
+
+    closePip() {
+        if (this.isHost) {
+            this.pipService.closePip();
+            return;
+        }
+        this._remoteAction({ pip: false });
+    }
```

Comment:
Nitpick: I'm fond of early-returns but for such one-liners branching, `if/else` is easier to read thanks to shape.